numpy/core/src/umath/loops.c.src

```diff
@@ -95,6 +97,28 @@
  * combine with NPY_GCC_OPT_3 to allow autovectorization
  * should only be used where its worthwhile to avoid code bloat
  */
+#define BASE_OUTPUT_LOOP(tout, op) \
+    OUTPUT_LOOP { \
+        tout * out = (tout *)op1; \
+        op; \
+    }
+#define OUTPUT_LOOP_FAST(tout, op) \
+    do { \
+    /* condition allows compiler to optimize the generic macro */ \
+    if (IS_OUTPUT_CONT(tout)) { \
+        if (args[0] == args[1]) { \
+            BASE_OUTPUT_LOOP(tout, op) \
+        } \
+        else { \
+            BASE_OUTPUT_LOOP(tout, op) \
+        } \
+    } \
+    else { \
+            BASE_OUTPUT_LOOP(tout, op) \
```

Comment:
Nit: If you're trying to match the unary case below, the indent should be one less on this line

Reply:
Thanks, fixed